### DISCUSSION TITLE
[!] Model file not found in the output path.

### DISCUSSION THREAD

**Discussion by TaterTotX** [coqui-ai/TTS#2361] [General Q&A]
I've been trying to download TTS on my M1 Mac for three days and finally succeeded. However, I keep getting an error message saying '[!] Model file not found in the output path.' Could someone please help me? Thank you very much.

from TTS.api import TTS
model_name = TTS.list_models()[0]
tts = TTS(model_name)
wav = tts.tts("This is a test! This is also a test!!", speaker=tts.speakers[0], language=tts.languages[0])

raise ValueError(" [!] Model file not found in the output path")
ValueError:  [!] Model file not found in the output path

**Comment by erogol** [comment]
Please do not double post
